src/xrpld/net/detail/RPCCall.cpp

```diff
@@ -1623,7 +1626,7 @@ fromNetwork(
             std::placeholders::_2,
             j),
         RPC_REPLY_MAX_BYTES,
-        RPC_NOTIFY,
+        RPC_WEBHOOK_TIMEOUT,
```

Comment:
This means it's giving up after 30 seconds of trying to deliver the webhook. E.g. the recipient receives the HTTP call but starts some processing, and doesn't respond... After 30 seconds, just give up instead of keeping the outbound HTTP connection open.

Reply:
Aha, got it. Does this code guaranteed to only be used with the webhook stuff or does this affect any other calls?